### PR TITLE
Refactor logo templates

### PR DESCRIPTION
**Pull request by kelvin-chappell**
This is an attempt to reduce the complexity of logo templates and isolate future problems a bit more.  Although a change to a css class will still affect them all.

/cc @Calanthe

### CODE REVIEW

File: common/app/views/fragments/commercial/cardLogo.scala.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+@(branding: common.commercial.Branding, isStandardSizeCard: Boolean)(implicit request: RequestHeader)
+
+<div class="badge @if(!isStandardSizeCard) {badge--branded}">
+    @branding.label
+    <a class="badge__link" href="@branding.sponsorLink" data-link-name="logo link">
+        <img src="@branding.sponsorLogo.url"
+             alt="@{branding.sponsorName}'s logo"
```
Comment (regiskuckaertz) [inline common/app/views/fragments/commercial/cardLogo.scala.html]: You can remove the `'s logo` part. The point of an alternative text is not to describe the image but to describe _the content_ of the image.

File: common/app/views/fragments/commercial/containerLogo.scala.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+@(branding: common.commercial.Branding, isOnTheLeft: Boolean)(implicit request: RequestHeader)
+@import common.commercial.PaidContent
+
+<div class="badge @if(isOnTheLeft){badge--alt}">
+    @branding.label
+    <a class="badge__link" href="@branding.sponsorLink" data-link-name="logo link">
+        <img src="@branding.sponsorLogo.url"
```
Comment (regiskuckaertz) [inline common/app/views/fragments/commercial/containerLogo.scala.html]: Shouldn't there be an AMP variant?

File: common/app/views/fragments/commercial/cardLogo.scala.html (hunk 1 above)
Comment (kelvin-chappell) [inline common/app/views/fragments/commercial/cardLogo.scala.html]: I see.  I always thought alt was to tell you what you were missing if you couldn't see the image.  I'll change that.

File: common/app/views/fragments/commercial/containerLogo.scala.html (hunk 2 above)
Comment (kelvin-chappell) [inline common/app/views/fragments/commercial/containerLogo.scala.html]: I thought amp was just for content pages.  But I'll add the amp variant.  So I suppose it should go back on the card logo as well.

**Pull request merged**